internal/service/alertconfiguration/model_alert_configuration.go

```diff
@@ -81,13 +81,13 @@ func NewMetricThreshold(tfMetricThresholdConfigSlice []TfMetricThresholdConfigMo
 	}
 }
 
-func NewMatcherList(list []TfMatcherModel) *[]map[string]any {
-	matchers := make([]map[string]any, len(list))
+func NewMatcherList(list []TfMatcherModel) *[]admin.StreamsMatcher {
+	matchers := make([]admin.StreamsMatcher, len(list))
 	for i, matcher := range list {
-		matchers[i] = map[string]any{
-			"fieldName": matcher.FieldName.ValueString(),
-			"operator":  matcher.Operator.ValueString(),
-			"value":     matcher.Value.ValueString(),
+		matchers[i] = admin.StreamsMatcher{
+			FieldName: matcher.FieldName.ValueString(),
+			Operator:  matcher.Operator.ValueString(),
+			Value:     matcher.Value.ValueString(),
```

Comment:
Only significant change was in alert configuration matchers model which is now typed (was a `map[string]any` before).